src/release/main.d

```diff
@@ -173,6 +172,40 @@ void main ( string[] params )
 }
 
 
+/*******************************************************************************
+
+    Creates a github release, including the associated milestone link at the top
+
+    Params:
+        con = connection to use
+        repo = repo to use
+        ver = version to release
+        notes = version notes
+
+*******************************************************************************/
+
+void createGithubRelease ( HTTPConnection con, Repository repo, string ver,
+                           string notes )
+{
+    import release.github;
+    import release.gitHelper;
+
+    import octod.api.releases;
+    import std.algorithm;
+    import std.stdio;
+    import std.range;
+
+    auto mstone_list = listMilestones(con, repo).find!(a=>a.title == ver);
+
+    string mstone_link;
+
+    if (!mstone_list.empty)
```

Comment:
Maybe report a warning if you find more than one (which could happen), or ideally let the user pick one (but might be too much effort for what it should be an error in neptune's ways).